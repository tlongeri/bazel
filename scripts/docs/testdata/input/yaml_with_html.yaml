book_path: /_book.yaml
project_path: /_project.yaml
title: Reference
landing_page:
  nav: left
  rows:
  - background: BAZEL_THEME
    options:
    - hero
    - description-50
    - no-image-background
    - padding-large
    items:
    - image_path: "/images/placeholder_960.png"
      heading: Reference
      description: >
        Look up commands, queries, and terminology necessary to working with <a href="/guide">Bazel</a>.