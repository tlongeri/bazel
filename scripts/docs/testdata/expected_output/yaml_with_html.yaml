book_path: /versions/6.6.6/_book.yaml
project_path: /_project.yaml
title: Reference
landing_page:
  nav: left
  rows:
  - background: BAZEL_THEME
    options:
    - hero
    - description-50
    - no-image-background
    - padding-large
    items:
    - image_path: "/versions/6.6.6/images/placeholder_960.png"
      heading: Reference
      description: >
        Look up commands, queries, and terminology necessary to working with <a href="/versions/6.6.6/guide">Bazel</a>.